src/server/controller/cycleData/getNodeValuesEstimations.ts

```diff
@@ -0,0 +1,27 @@
+import { CountryIso } from 'meta/area'
+import { Assessment, Cycle, NodeValuesEstimation } from 'meta/assessment'
+
+import { BaseProtocol, DB } from 'server/db'
+import { TableRepository } from 'server/repository/assessment/table'
+import { NodeValueEstimationRepository } from 'server/repository/assessmentCycle/nodeValueEstimationRepository'
+
+export const getNodeValuesEstimations = async (
+  props: {
+    assessment: Assessment
+    countryIso: CountryIso
+    cycle: Cycle
+    tableName: string
+  },
+  client: BaseProtocol = DB
+): Promise<Record<string, NodeValuesEstimation>> => {
+  const { assessment, cycle, countryIso, tableName } = props
+
+  const table = await TableRepository.getOne({ assessment, cycle, tableName })
```

Comment:
getOne should return the table. table?.uuid should be table.uuid. can u please verity tableRepository.getOne ?

Reply:
you're right. at some point I modified the tableRepository.getOne because it can't return null (it returns an error if table doesn't exist). the null check is useless there and I removed it.